nixos/modules/services/mail/dovecot.nix

```diff
@@ -4,14 +4,67 @@ let
   inherit (lib) any attrValues concatMapStringsSep concatStrings
     concatStringsSep flatten imap1 isList literalExpression mapAttrsToList
     mkEnableOption mkIf mkOption mkRemovedOptionModule optional optionalAttrs
-    optionalString singleton types;
+    optionalString singleton types mkRenamedOptionModule nameValuePair
+    mapAttrs' listToAttrs match filter;
```

Comment:
lib.match doesn't seem to exist, not in master nor this commit.

```
% nix eval 'github:RaitoBezarius/nixpkgs/new-sieve-dovecot-api#lib.match'
error: flake 'github:RaitoBezarius/nixpkgs/new-sieve-dovecot-api' does not provide attribute 'packages.x86_64-linux.lib.match', 'legacyPackages.x86_64-linux.lib.match' or 'lib.match'
       Did you mean path?
```

Reply:
Yeah, it's `lib.strings.match` unfortunately…

Reply:
Fixed.